### ISSUE TITLE
Discussion: Adding new maintainer

### ISSUE THREAD

**Issue by gleich** [fluttercommunity/community#102]
# Discussion: Adding new maintainer

I need to add @vkammerer as an outside collaborator to the `import_sorter` repo.

**Comment by vkammerer** [comment]
@jeroen-meijer : @gleich agreed that I can be a maintainer of `import_sorter`. Could you please give me the relevant permissions / access to the repo?

**Comment by vkammerer** [comment]
@gleich it's been almost a month now, two messages and still no reply from @jeroen-meijer . Would there be anyone else that we could contact to add me as a maintainer of `import_sorter` ?

**Comment by mhadaily** [comment]
Welcome the the flutter community package maintainer. You are now maintainer of import_sorter package @vkammerer
